stdlib/2and3/contextlib.pyi

```diff
@@ -64,3 +64,29 @@ if sys.version_info >= (3,):
         def pop_all(self) -> ExitStack: ...
         def close(self) -> None: ...
         def __enter__(self: _U) -> _U: ...
+
+if sys.version_info >= (3, 7):
+    from asyncio.futures import Future
```

Comment:
Why not use typing.Awaitable instead of Future?

Reply:
I was looking at stubs for `asyncio.tasks` as an example.